bolides/bdf.py

```diff
@@ -25,6 +29,36 @@
                'impact-e', 'alt', 'vel']
 
 utc = timezone('UTC')
+def haversine(lat1, lon1, lat2, lon2):
+        """
+        Calculates the haversine distance between two points in km.
+        Inputs are in degrees.
+        """
+        R = 6371  # Earth radius in km
+
+        # Convert degrees to radians
+        lat1, lon1, lat2, lon2 = map(np.radians, [lat1, lon1, lat2, lon2])
+
+        dlat = lat2 - lat1
+        dlon = lon2 - lon1
+
+        a = np.sin(dlat / 2)**2 + np.cos(lat1) * np.cos(lat2) * np.sin(dlon / 2)**2
+        c = 2 * np.arcsin(np.sqrt(a))
+        return R * c
+def distance_metric(x, y, fov_goes, total_time, n):
+    """
+    Calculates the distance between two points considering time
+    ----------------------------
```

Comment:
Also... this is a type of distance metric, but it's really a measurement of how improbable it is two bolides are so close together in time and space. There are generalized distance metrics where you can set the relative weighting between the terms.  I showed an example formula near the beginning of this summer. 
Although this is a good and useful formula, we should be sure to say what this formula is actually measuring. We could then have an option to use a more standardized generalized distance metric. The formula would allow for the user to specify the relative weighting between the terms. Perhaps this would be a second pull request (and it's not necessary for you to write it).